javascript/src/main/pages/Admin/ViewLogins.js

```diff
@@ -29,6 +29,8 @@ const ViewLogins = () => {
     <>
       <h2 className="text-left">Login History</h2>
       <Button><CSVLink style={{color: "white"}} headers={headers} data={loginTable || []} filename = {"LoginTable.csv"}>Download CSV</CSVLink></Button>
+      <br></br>
```

Comment:
I worked on something similar in one of the starter stories. Instead of using line breaks, I changed the margin. I was wondering if this should be implemented that way?